src/edu/washington/escience/myria/operator/DbCreateUDF.java

```diff
@@ -0,0 +1,75 @@
+/**
+ *
+ */
+package edu.washington.escience.myria.operator;
+
+import com.google.common.collect.ImmutableMap;
+
+import edu.washington.escience.myria.DbException;
+import edu.washington.escience.myria.MyriaConstants;
+import edu.washington.escience.myria.accessmethod.AccessMethod;
+import edu.washington.escience.myria.accessmethod.ConnectionInfo;
+import edu.washington.escience.myria.storage.TupleBatch;
+
+/**
+* 
+*/
+public class DbCreateUDF extends RootOperator {
+  /** Required for Java serialization. */
+  private static final long serialVersionUID = 1L;
+
+  /** The connection to the database database. */
+  private AccessMethod accessMethod;
+  /** The information for the database connection. */
+  private ConnectionInfo connectionInfo;
+
+  private final String udfDefinition;
+
+  /**
+   * @param child the source of tuples to be inserted.
+   * @param relationKey the key of the table the tuples should be inserted into.
+   * @param connectionInfo the parameters of the database connection.
+   */
+  public DbCreateUDF(final Operator child, final String sqlString, final ConnectionInfo connectionInfo) {
+    super(child);
+    this.connectionInfo = connectionInfo;
+    udfDefinition = sqlString;
+  }
+
+  @Override
+  protected void init(final ImmutableMap<String, Object> execEnvVars) throws DbException {
+    /* Retrieve connection information from the environment variables, if not already set */
+    if (connectionInfo == null && execEnvVars != null) {
+      connectionInfo = (ConnectionInfo) execEnvVars.get(MyriaConstants.EXEC_ENV_VAR_DATABASE_CONN_INFO);
+    }
+    /* Open the database connection */
+    accessMethod = AccessMethod.of(connectionInfo.getDbms(), connectionInfo, false);
+
+    /* Drop the table */
+    accessMethod.runCommand(udfDefinition);
```

Comment:
By "generic" ```DbExecute``` operator do you mean having another new operator exclusively handle UDF creation? 

We can handle it like we did with DbCreateView/DbCreateIndex where we asked for specific input parameters. 

Or we could just add some sort of validation before calling "runCommand". If we did validate, what would we check for? Some things I could think of include: 1) checking for "CREATE FUNCTION" keyword 2) checking if there is no additional command running after the UDF creation command